src/Main.tsx

```diff
@@ -51,6 +54,7 @@ const Main: React.FC<Props> = ({
     <ApolloProvider client={apolloClient}>
       <EditorApp
         fileManagement={fileManagement}
```

Comment:
the file management is technically a part of submission right? This is because it's related to submission and I don't think it is possible to have fileManagement without submission so maybe it would be better to have it inside the submission object. I'm also concerned here about not creating to much properties on the article-editor. But I'm not insisting, maybe it's ok.

What I think we definitely need to do is to make sure that passing fileManagement is optional on the article editor.

Reply:
> What I think we definitely need to do is to make sure that passing fileManagement is optional on the article editor.

without `fileManagement` we can't use or open the file section in the editor inspector, and as far as I know file section will be mandatory for any app!

and I'm going to remove the `submissionId` as we can get the id from the submission object.

Reply:
ah, you're right!